docs/source/release_notes/release-notes.rst

```diff
@@ -52,6 +52,12 @@ develop
            Atlas users should use ``AtlasDbMetrics.setMetricRegistry`` to set a ``MetricRegistry``.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1429>`__)
 
+    *    - |new|
+         - There is now a ``TimestampMigrationService`` with the ``fast-forward`` method that can be used to migrate between timestamp services.
+           You will simply need to fast-forward the new timestamp service using the latest timestamp from the old service.
+           This can be done using the :ref:`timestamp forward cli <offline-clis>` when your AtlasDB services are offline.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/?>`__)
```

Comment:
Hmm. This is true, but I'd be a little cautious with publicising this now. I'm uncertain if Timelock will definitely be in v0.28.0, and the upgrade-downgrade workflow that's being used on internal product might cause data corruption if the necessary steps aren't taken.

It's true that this does make it easier to do Persistent -> Persistent migrations though; @rhero any thoughts?